vowpalwabbit/continuous_actions_reduction_features.h

```diff
@@ -33,6 +33,17 @@ struct reduction_features
     pdf.clear();
     chosen_action = std::numeric_limits<float>::quiet_NaN();
   }
+
+  void check_valid_pdf_or_clear()
```

Comment:
Should this be a member function? Seems like it should be external and reduction_features stays as a simple container